packages/backend-app-api/src/services/implementations/auth/user/UserTokenHandler.ts

```diff
@@ -137,7 +137,6 @@ export class UserTokenHandler {
       base64url.encode(
         JSON.stringify({
           sub: payload.sub,
-          ent: payload.ent,
```

Comment:
Just noting that this upgrade means that limited tokens that were issued, will no longer be accepted. You may want to make noise in the changeset that after this change, users may have to clear their cookies in their browsers.

Reply:
Good point!